### PR TITLE
Move human_in_the_loop.py example to zoo

### PR DESCRIPTION
**Pull request by qianyi-sun**
#1220

### CODE REVIEW

**Comment by Gamenot** [comment]
You will want to run `black` formatting.

File: zoo/evaluation/human_in_the_loop.py [hunk 1]
```diff
@@ -22,7 +22,7 @@
 if __name__ == "__main__":
     from argument_parser import default_argument_parser
 else:
-    from .argument_parser import default_argument_parser
+    from examples.argument_parser import default_argument_parser
```
Comment (Gamenot) [inline zoo/evaluation/human_in_the_loop.py:25]: This import should be removed.

File: zoo/evaluation/human_in_the_loop.py [hunk 2]
```diff
@@ -22,7 +22,7 @@
 if __name__ == "__main__":
     from argument_parser import default_argument_parser
 else:
```
Comment (Gamenot) [inline zoo/evaluation/human_in_the_loop.py:24]: Also this import.

**Review by Gamenot — CHANGES_REQUESTED** [review]
You will need to list this agent in `zoo/policies/__init__.py`.

**Comment by Gamenot** [comment]
Please run `black`.

**Comment by qianyi-sun** [comment]
> Please run `black`.
reformatted

File: zoo/policies/human_in_the_loop.py [hunk 3]
```diff
@@ -17,13 +17,6 @@
...
-# The following ugliness was made necessary because the `aiohttp` #
-# dependency has an "examples" module too.  (See PR #1120.)
-if __name__ == "__main__":
-    from argument_parser import default_argument_parser
-else:
-    from .argument_parser import default_argument_parser
-
 logging.basicConfig(level=logging.INFO)
 
 AGENT_ID = "Agent-007"
```
Comment (Gamenot) [inline zoo/policies/human_in_the_loop.py:22]: These lines can be removed as well.
```suggestion
```

File: zoo/policies/human_in_the_loop.py [hunk 4]
```diff
@@ -17,13 +17,6 @@
 from smarts.core.agent_interface import AgentInterface, AgentType
 from smarts.core.utils.episodes import episodes
```
Comment (Gamenot) [inline zoo/policies/human_in_the_loop.py:18]: This import can also be removed.
```suggestion
```

Please also remove the `gym` and `logging` imports.

**Review by Gamenot — APPROVED** [review]
What is left is nitpick that is not blocking but good to clean up.

**Review by RutvikGupta — APPROVED** [review]

**Pull request merged**